src/Ocelot/LoadBalancer/LoadBalancers/CookieStickySessionsCreator.cs

```diff
@@ -14,8 +15,9 @@ public Response<ILoadBalancer> Create(DownstreamRoute route, IServiceDiscoveryPr
         {
             var loadBalancer = new RoundRobin(async () => await serviceProvider.Get());
             var bus = new InMemoryBus<StickySession>();
+            var sessionStorage = new InMemoryStickySessionStorage();

```

Comment:
Is it possible to inject this default concrete class from DI-container, and/or via extra argument?
Because this method is responsible for default implementations injection, we need to forward a `IStickySessionStorage` object to `CookieStickySessions` constructor.

---
> substitute CookieStickySessionsCreator to inject into CookieStickySessions own storage mechanism

Because [this class (creator) is managed by DI-container](https://github.com/ThreeMammals/Ocelot/blob/develop/src/Ocelot/DependencyInjection/OcelotBuilder.cs#L114), we have to add  `InMemoryStickySessionStorage` class to DI-container too, and inject it via constructor. This is the simplest solution!
Finally, developer should not implement new creator inheriting the `ILoadBalancerCreator` interface.